pkg/resources/operatingsystemmanager/deployment.go

```diff
@@ -56,7 +56,7 @@ var (
 
 const (
 	Name = "operating-system-manager"
-	Tag  = "28d391f034d886bf5a3aad5eb0b2abec99e5c1b6"
+	Tag  = "xrstf4"
```

Comment:
```suggestion
	Tag  = "34aa7a21be09e052cde6bffd8b54ca25e4e243b8"
```

Reply:
WTF happened during my last rebase?! I replaced the xrstf4 already. Damn you, git..